tests/shared/src/test/scala/epollcat/TcpSuite.scala

```diff
@@ -37,6 +37,8 @@ import scala.concurrent.duration._
 
 class TcpSuite extends EpollcatSuite {
 
+  override def munitIOTimeout = 10.seconds
```

Comment:
Lee, what do you suppose is the timeout we should use? The default is 30 seconds but I worried if that is too long for the impatient!

Our tests run quickly so we could make this even smaller I think.

Reply:
Thank you for this change. I think it will well repay its cost.
(Easy for me to say, I did not pay the cost.)

I think you are asking a classic "receiver operating characteristic" (ROC) curve question. The balance
of cost between failing too soon and of waiting too long.

I think the overall answer is "pick a number and 
experience will let you know if you are right".  Helpful,
aren't I?

Is the timeout per-test or per-Suite?  I am still learning
munit.

If it is per-test, my suggestion would be to bump to 
20 seconds or 15 seconds. On the happy path, the tests
should be instantaneous.  Here we are hedging on error 
paths. They tend to be bi-modal, either instantaneous or
 delayed more than one would expect (i.e. perceived hang).   

If the happy path is the more likely, the amortized 
execution time should be short (1 sec?) no matter what
the "worst case" is.    

In school when I was learning to graphical applicatons,
where a real user was spending clock time looking at the 
screen, I was told that for anything over 2 seconds response
you have lost the persons attention.  I think students these
days are taught something like (0.5) seconds or less.

Over the past few years, I have had multiple complaints
about the "short timing" of several SN ProcessTest test
cases.  I did not write the original code, but have been 
catching the complains, and trying to fix them. Making
the timeout on the generous side, but not 30 or 127 seconds,
will cut down on your/our maintenance costs without
dissatisfying most users.

The two places which tend to be a lot slower than one would
expect, by an order of magnitude or more, are GitHub Continuous Integration and Windows code.  My current working hypothesis is that there are context switches happening.
Yes, the "wall clock" should stop in a VM when it swaps out,
but I could paper my walls with ProcessTest complaints.

Reply:
> If it is per-test, my suggestion would be to bump to
> 20 seconds or 15 seconds.

Yes, the timeout is per-test. I've set it to 20 seconds for now. Feel free to adjust it as you continue to work :)

Thanks for your input! 😁